### PR TITLE
(REFACTOR) find success state w/ form elements

### PR DESCRIPTION
**Pull request by sfount**
This pull request implements the loaded state of the find patient directive using only standard form elements - this allows the directive to take up the same space on completion as well as provide a slightly more uniform interface for refreshing. 

_Current success state_
Before

_Proposed changes_
Proposed success

Proposed failure

**Prototype - additional information**
This demonstrates the ability to be able to display additional contextual information about the patient without taking up additional space. This is just a prototype and can be adapted or removed following team discussion. 

Currently this information is shown on a `mouseenter` event, allowing the user to highlight the `i` icon to view more details about the patient. 

Context

Popover prototype

### CODE REVIEW

File: client/src/partials/templates/findpatient.tmpl.html [hunk 1]
```diff
@@ -53,7 +57,13 @@
       <!-- submit button for search by ID -->
       <!-- use of fixed width that correspond to notification text margin right -->
       <span class="input-group-btn">
+    
+        <!-- 
+         button type="button" allows angular to disable the elements click event as well
+         as not submitting the form
```
Comment (jniles) [inline client/src/partials/templates/findpatient.tmpl.html:63]: Good to know.  Thanks for the comment!

File: client/src/partials/templates/findpatient.tmpl.html [hunk 2]
```diff
@@ -72,24 +82,34 @@
...
+
+        <input ng-if="session.loadStatus=='loaded'" readonly class="form-control" value="[{{patient.reference}}] {{patient.name}}">
+        <input ng-if="session.loadStatus=='error'" readonly class="form-control" value="{{ 'FIND.PATIENT_NOT_FOUND' | translate }}">
+        
+        <div class="input-group-btn">
+          <a 
+            ng-if="session.loadStatus=='loaded'"
+            class="btn btn-default"
+            uib-popover-template="'partials/templates/popover/patientinfo.tmpl.html'"
+            popover-title="Patient Record"
```
Comment (jniles) [inline client/src/partials/templates/findpatient.tmpl.html:102]: This will need translating down the line.  No reason to block this PR though.

**Comment by jniles** [comment]
This PR is a nice prototype with good code and comments.  It will be easily merged.

We should discuss as a team how to display more information about a patient from the `find-patient` component.  The idea to show more information is excellent.  This popover is a good demonstration of its usefulness.

**Comment by sfount** [comment]
Thanks for the comments! 

It should be discussed by the team and iterated as many times as needed! 

A few ideas that could be explored : 
- If this information is needed all the time it should be displayed all the time 
- If this information is only needed from time to time a popover may be appropriate 
- If more information is needed then it may be more appropriate to used a modal component allowing us to show debtor history etc.

This could be proposed as an issue, what does everyone think?

**Comment by jniles** [comment]
Transitioning to an issue is a good idea.  Go ahead and make one.

**Pull request merged**